supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc

```diff
@@ -149,7 +149,7 @@
 [discrete]
 [[window-maker]]
 ==== image:images/yes.png[yes] Window Maker (noun)
-*Description*: "Window Maker" is a window manager for the X Window System. Do not combine Window Maker into one word or hyphenate the two words.
+*Description*: _Window Maker_ is a window manager for the X Window System. Do not combine Window Maker into one word or hyphenate the two words.
```

Comment:
Do not combine "Window Maker" into one word or hyphenate the two words.
Write Preview